templates/connect-inject-clusterrole.yaml

```
@@ -17,4 +17,12 @@ rules:
     - "list"
     - "watch"
     - "patch"
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  resourcesName:
+  - {{ template "consul.fullname" . }}-connect-injector-webhook
```

To match the name defined in the pod security policy:
```suggestion
  - {{ template "consul.fullname" . }}-connect-injector-webhook-deployment
```